### PR TITLE
doc: Add the sd-jwt vc description in sdjwt.js.org

### PR DESCRIPTION
**Pull request by lukasjhan**
I added description of sd-jwt vc to our homepage.
I wrote a simple one based on what @cre8 wrote in the readme. :)

If there is anything you would like to write more about, please let me know.

### CODE REVIEW

**Comment by cre8** [comment]
Maybe we should also add the package to the readme in the root folder.

And later a part "who is using sd-jwt" and link there the Veramo and credo integration

**Comment by lukasjhan** [comment]
> Maybe we should also add the package to the readme in the root folder.
> 
> And later a part "who is using sd-jwt" and link there the Veramo and credo integration

I'll add it right away

File: README.md [hunk 1]
```diff
@@ -10,7 +10,10 @@ This is the reference implmentation of [IETF SD-JWT specification](https://data
 
 Hopae, a founding member of OpenWallet Foundation, is building wallet module in TypeScript and need this project as a core component.
 
-Currently compliant with: **draft-ietf-oauth-selective-disclosure-jwt-07**
+Currently compliant with:
+
+- **draft-ietf-oauth-selective-disclosure-jwt-07**
```
Comment (cre8) [inline README.md:15]:
```suggestion
- **[draft-ietf-oauth-selective-disclosure-jwt-07](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/)**
```

File: README.md [hunk 2]
```diff
@@ -10,7 +10,10 @@ This is the reference implmentation of [IETF SD-JWT specification](https://data
...
+- **draft-ietf-oauth-sd-jwt-vc-02**
```
Comment (cre8) [inline README.md:16]:
```suggestion
- **[draft-ietf-oauth-sd-jwt-vc-02](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html)**
```

**Review by cre8 — CHANGES_REQUESTED** [review]

**Comment by lukasjhan** [comment]
@cre8 I updated README with your suggestion :)

**Review by cre8 — APPROVED** [review]

**Pull request merged**